tests/state/test_get_list_state.py

```diff
@@ -0,0 +1,365 @@
+        '--reverse',
+    ])
+
+    expected_address = json.loads(result.output).get('result')[0].get('address')
+
+    assert PASSED_EXIT_FROM_COMMAND_CODE == result.exit_code
+    assert expected_address == ADDRESS_WITH_STATE
+
+
+def test_get_list_state_with_address():
+    """
+    Case: get a state by its address.
+    Expect: state is returned.
+    """
+    runner = CliRunner()
+    result = runner.invoke(cli, [
+        'state',
+        'get-list',
+        '--address',
+        ADDRESS_WITH_STATE,
+        '--node-url',
+        NODE_IP_ADDRESS_FOR_TESTING,
+    ])
+
+    expected_address = json.loads(result.output).get('result')[0].get('address')
+
+    assert PASSED_EXIT_FROM_COMMAND_CODE == result.exit_code
+    assert expected_address == ADDRESS_WITH_STATE
+
+
+@pytest.mark.parametrize('flag', ['--address', '--start'])
+def test_get_list_state_with_invalid_address_start(flag):
+    """
+    Case: get a list of states by its invalid address and account address starting from.
+    Expect: the following address is invalid error message.
+    """
+    invalid_address = '044c7db163cf2'
+
+    runner = CliRunner()
+    result = runner.invoke(cli, [
+        'state',
+        'get-list',
+        flag,
+        invalid_address,
+        '--node-url',
+        NODE_IP_ADDRESS_FOR_TESTING,
+    ])
+
+    expected_error_message = {
+        'errors': {
+            'address': [
+                f'The following address `{invalid_address}` is invalid.',
+            ],
+        },
+    }
+
+    assert FAILED_EXIT_FROM_COMMAND_CODE == result.exit_code
+    assert dict_to_pretty_json(expected_error_message) in result.output
+
+
+@pytest.mark.parametrize('flag', ['--address', '--start'])
+def test_get_list_state_with_non_existing_address_start(flag):
+    """
+    Case: get a state by its non-existing address and account address starting from.
+    Expect: block not found error message.
+    """
+    non_existing_address = '0000000000000000000000000000000000000000000000000000000000100000000031'
+
+    runner = CliRunner()
+    result = runner.invoke(cli, [
+        'state',
+        'get-list',
+        flag,
+        non_existing_address,
+        '--node-url',
+        NODE_IP_ADDRESS_FOR_TESTING,
+    ])
+
+    expected_error_message = {
+        'errors': f'Block not found.',
+    }
+
+    assert FAILED_EXIT_FROM_COMMAND_CODE == result.exit_code
+    assert dict_to_pretty_json(expected_error_message) in result.output
+
+
+def test_get_list_state_with_start():
+    """
+    Case: get a list states by account address starting from.
+    Expect: list of states is returned.
+    """
+    runner = CliRunner()
+    result = runner.invoke(cli, [
+        'state',
+        'get-list',
+        '--start',
+        ADDRESS_WITH_STATE,
+        '--node-url',
+        NODE_IP_ADDRESS_FOR_TESTING,
+    ])
+
+    result_data = json.loads(result.output).get('result')[0]
+
+    assert PASSED_EXIT_FROM_COMMAND_CODE == result.exit_code
+    assert result_data
```

Comment:
Add check.

Reply:
Why your check is better? My check universality and does not depend on network uptime.
Give me, please, use-case where my check can break, and how I can test it.